recipes/pycoal/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pycoal" %}
+{% set version = "0.5.1" %}
+{% set sha256 = "d066af5e92d1d8250d972c7a683f70403df40e2aa3bac2d45655b301df7c7a18" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - spectral
+
+test:
+  imports:
+    - pycoal
+
+about:
+  home: https://github.com/capstone-coal/pycoal/
+  license: Apache 2.0
+  license_family: Apache
+  license_file: LICENSE
```

Comment:
In addition, GPLv2 is [not automatically compatible](https://www.gnu.org/licenses/license-list.html#GPLv2) with GPLv3:
> GNU General Public License (GPL) version 2 (#GPLv2)
>
> This is the previous version of the GNU GPL: a free software license, and a copyleft license. We recommend the latest version for most software.
>
> Please note that GPLv2 is, by itself, not compatible with GPLv3. However, most software released under GPLv2 allows you to use the terms of later versions of the GPL as well. When this is the case, you can use the code under GPLv3 to make the desired combination. To learn more about compatibility between GNU licenses, please see our FAQ.
I don't know if Spectral Python has an "any later version" clause.